packages/site/src/lib/helpers/glosses.test.ts

```diff
@@ -62,6 +62,64 @@ describe('orderGlosses', () => {
   });
 });
 
+describe('orderGlosses with multiple examples', () => {
+  const $t = (id: string) => {
```

Comment:
@jacob-8 could I use general constants in tests if I'm reusing code?

Reply:
I don't understand the question, can you please explain further?